yamllint/rules/document_start.py

```diff
@@ -79,8 +79,12 @@
 
 ID = 'document-start'
 TYPE = 'token'
-CONF = {'present': bool}
-DEFAULT = {'present': True}
+CONF = {'present': bool,
+        'max-empty-lines-after': (int, -1),
+        'min-empty-lines-after': (int, 0)}
+DEFAULT = {'present': True,
+           'max-empty-lines-after': -1,
+           'min-empty-lines-after': 0}
```

Comment:
You also need to update documentation, see lines 18-71 above ↑ :wink: 
(no need to add examples in my opinion, just two new bullet points inside *Options*)

You can take inspiration in other rules, for instance:
https://github.com/adrienverge/yamllint/blob/4374490/yamllint/rules/commas.py#L22-L27

Reply:
Done.